packages/react-dom/src/events/plugins/SimpleEventPlugin.js

```diff
@@ -108,6 +115,18 @@ function extractEvents(
     case 'contextmenu':
       EventInterface = MouseEventInterface;
       break;
+    case 'dblclick':
+      if (!inCapturePhase && prevBubblePhaseEvent === nativeEvent) {
```

Comment:
This won't work with multiple React's on the same page though right? For that case we could attach a property to the event object so other React's know to also not process the event.

Reply:
I think it's the other way around: we *would* want the outer React to process the event because the inner React wouldn't include the outer tree's nodes in the traversal. However, the outer tree would also want to only handle it the first time.

Reply:
(The test verifies this is what happens)

Reply:
That makes sense.